src/dmd/traits.d

```diff
@@ -1418,18 +1418,29 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
         auto o1 = (*e.args)[1];
 
         FuncDeclaration fd;
-        TypeFunction tf = toTypeFunction(o, fd);
-
+        Expression exp = isExpression(o);
+        CallExp ce;
         ParameterList fparams;
-        if (tf)
-            fparams = tf.parameterList;
-        else if (fd)
-            fparams = fd.getParameterList();
+
+        if(exp)
+            ce = exp.isCallExp();
```

Comment:
`exp` is only needed once (and we use a space after `if`):
```
CallExp ce;
if (auto exp = isExpression(o))
    ce = exp.isCallExp();
```

Reply:
I always forget the spaces :sweat_smile: . Done